data-migration/main.tf

```diff
@@ -0,0 +1,92 @@
+resource "aws_iam_role" "s3_migration_role" {
+  name               = local.role_name
+  assume_role_policy = data.aws_iam_policy_document.allow_assume_role.json
+}
+
+data "aws_iam_policy_document" "allow_assume_role" {
+  statement {
+    effect = "Allow"
```

Comment:
```suggestion
    sid    = "AllowAssumeWorkerRole"
    effect = "Allow"
```

Reply:
Why are these resolved without comment or accepting?

Reply:
ooh, maybe because they were marked as outdated?  Does that automatically mark them as resolved?  I certainly didn't resolve them.  Will go through now.

Reply:
Not sure 🤔